joplin/js/CreateContentModal/index.js

```diff
@@ -190,6 +191,7 @@ class CreateContentModal extends Component {
                           this.handleContentOrTopicSelect
                         }
                         content_or_topic={this.state.content_or_topic}
+                        canSetTopics={this.state.can_set_topics}
```

Comment:
Let's keep CamelCase for js whenever possible.

Reply:
fixed.